ConsoleGameLib/Helpers/MathHelper.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace ConsoleGameLib.Helpers
+{
+    public static class MathHelper
+    {
+        public static float Clamp(float min, float max, float val)
+        {
+            if(val > max)
+            {
+                return max;
+            }
+            if(val < min)
+            {
+                return min;
+            }
+            return val;
+        }
+        public static float ClampMin(float min, float val)
+        {
```

Comment:
Per discussion with Peter, the idea is a one-side capping of the value, rather than automatic return of a max value, as Math.Max would do.